src/data/directory/lgu/negros-island-region.json

```
@@ -652,10 +652,11 @@
           "municipality": "Maria",
           "zip_code": "6229",
           "mayor": {
-            "name": "ROSELYN T. ASOK"
+            "name": "ROSELYN T. ASOK",
+            "contact": "(032) 542-6539"
           },
           "vice_mayor": {
-            "name": "IVY DAN M. SAMSON"
+            "name": "MEYNARD ASOK"
```

```suggestion
            "name": "MEYNARD R. ASOK"
```

ref: https://lgu201.dilg.gov.ph/view.php?r=07&p=061&m=04